code/modules/vehicles/scooter.dm

```diff
@@ -206,6 +206,21 @@
 			to_chat(rider, span_warning("[src] [p_are()] not powerful enough to fly upwards."))
 		return FALSE
 
+/obj/vehicle/ridden/scooter/skateboard/hoverboard/holyboarded
+	name = "holy skateboard"
+	desc = "A board blessed by the gods with the power to grind for our sins. Has the initials 'J.C.' on the underside."
+	board_item_type = /obj/item/melee/skateboard/holyboard
+	instability = 3
+	icon_state = "hoverboard_holy"
+/obj/vehicle/ridden/scooter/skateboard/hoverboard/holyboarded/post_buckle_mob(mob/living/M)
+	M.AddComponent(/datum/component/anti_magic, MAGIC_RESISTANCE|MAGIC_RESISTANCE_HOLY)
+	return ..()
+
+/obj/vehicle/ridden/scooter/skateboard/hoverboard/holyboarded/post_unbuckle_mob(mob/living/M)
+	if(!has_buckled_mobs())
+		qdel (M.GetComponent(/datum/component/anti_magic, MAGIC_RESISTANCE|MAGIC_RESISTANCE_HOLY))
```

Comment:
isn't this handled by the component itself?
```
/datum/component/anti_magic/proc/on_drop(atom/movable/source, mob/user)
	SIGNAL_HANDLER

	// Reset alert
	if(source.loc != user)
		alert_caster_on_equip = TRUE
	unregister_antimagic_signals(user)
```

Reply:
this would work on an item, however this is a vehicle, so i just decided to play it safe and have it manually delete the component since there is no item to hold the component. Basically, i added it without an item, so i have to remove it without using the item.

Reply:
Do you think you'd be able to make the antimagic component compatible with ridden vehicles, by having the antimagic signals registered to the buckled mob and viceversa when unbuckled?

Reply:
if a mob with anti-magic (that hasn't been in the game yet by the way) happens to ride this holy skateboard and gets their powers reversed, i will make sure to fix that in the future.

Reply:
Oh, well, I'll do it myself then...